code/datums/flock/flock.dm

```diff
@@ -447,11 +447,11 @@ var/flock_signal_unleashed = FALSE
 /datum/flock/proc/getActiveTraces()
 	var/list/active_traces = list()
 	for (var/mob/living/intangible/flock/trace/T as anything in src.traces)
-		if (T.client)
+		if (T.client && T.afk_counter < T.afk_counter_threshold)
```

Comment:
you're only updating `afk_counter` for flockminds, not traces, so this doesn't make sense